libqfieldsync/offline_converter.py

```diff
@@ -102,7 +103,8 @@ def __init__(
         # elipsis workaround
         self.trUtf8 = self.tr
 
-        self.export_folder = Path(export_folder)
+        self._export_filename = Path(export_filename)
```

Comment:
```suggestion
        if not export_filename:
        	raise Exception("Empty export filename provided!")

        self._export_filename = Path(export_filename)
```

We need to be sure it is an existing path.